datahub-web-react/src/app/identity/user/SelectRole.tsx

```diff
@@ -23,16 +23,14 @@ const RoleIcon = styled.span`
 `;
 
 export default function SelectRole({ user, userRoleUrn, selectRoleOptions, refetch }: Props) {
-    const [isViewingAssignRole, setIsViewingAssignRole] = useState(false);
-    const [roleToAssign, setRoleToAssign] = useState<DataHubRole>();
-
+    const noRoleText = 'No Role';
```

Comment:
tiny nit: but usually constants like this can be defined outside of the component to clean things up in here, so like higher up in the file defining `NO_ROLE_TEXT = 'No Role';`

Reply:
+1

Reply:
Good call, refactored!